specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.benefits.json

```diff
@@ -0,0 +1,1002 @@
+        "description": "Lists benefit utilization summaries for the provided billingAccount. This API provides benefit utilization summaries only for enrollment accounts.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/cost-management/"
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/billingAccountIdParameter"
+          },
+          {
+            "$ref": "#/parameters/grainParameter"
+          },
+          {
+            "name": "filter",
+            "in": "query",
+            "description": "Supports filtering by benefitType, benefitId, benefitOrderId and usageDate. benefitType is defaulted to savingsPlan.",
+            "required": false,
+            "type": "string",
+            "x-ms-parameter-location": "method"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/BenefitUtilizationSummariesListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common-types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "SavingsPlanUtilizationSummaries-BillingAccount": {
+            "$ref": "./examples/BenefitUtilizationSummaries/SavingsPlan-BillingAccount.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}/providers/Microsoft.CostManagement/benefitUtilizationSummaries": {
+      "get": {
+        "tags": [
+          "BenefitUtilizationSummaries"
+        ],
+        "operationId": "BenefitUtilizationSummaries_ListByBillingProfileId",
+        "description": "Lists benefit utilization summaries for billingProfile.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/cost-management/"
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/billingAccountIdParameter"
+          },
+          {
+            "$ref": "#/parameters/billingProfileIdParameter"
+          },
+          {
+            "$ref": "#/parameters/grainParameter"
+          },
+          {
+            "name": "filter",
+            "in": "query",
+            "description": "Supports filtering by benefitType, benefitId, benefitOrderId and usageDate. benefitType is defaulted to savingsPlan.",
```

Comment:
Should it be modeled as an enum? Since its a finite list of values you're allowing.

Reply:
Not addressing this suggestion, since I want to keep the filter definition consistent with Cost management RP. Also, in the code we plan to support using ODataQueryOptions. So in the future, we may end up supporting filters on all the properties.